test/angular-view.test.js

```diff
@@ -0,0 +1,197 @@
+          })
+          .on('ready', function (generator) {
+            // this is called right before `generator.run()` is called
+          })
+          .on('end', function () {
+            done();
+          });
+      });
+
+      it('should generate an angular view file', function () {
+        assert.file('modules/core/client/config/core.client.routes.js');
+        assert.file('modules/core/client/views/core.client.view.html');
+
+        assert.fileContent('modules/core/client/config/core.client.routes.js',
+          "'use strict';\n\n" +
+          "//Setting up route\n" +
+          "angular.module('core').config(['$stateProvider',\n" +
+          "  function($stateProvider) {\n" +
+          "    // Core state routing\n" +
+          "    $stateProvider.\n" +
+          "      state('core', {\n" +
+          "        url: '/fooroute',\n" +
+          "        templateUrl: 'modules/core/client/views/core.client.view.html'\n" +
+          "      });\n" +
+          "  }\n" +
+          "]);\n"
+        );
+      });
+    });
+
+    describe('Add a route path with route file already existing', function () {
+      beforeEach(function (done) {
+        helpers.run(path.join(__dirname, '../angular-view'))
+          .withOptions({
+            'skip-install': true
+          })
+          .withPrompts({
+            'moduleName': 'core',
+            'name': 'bar',
+            'controllerName': 'barctrl',
+            'addRoute': true,
+            'routePath': 'barroute'
+          })
+          .on('ready', function (generator) {
+            mkdirp.sync('modules/core/client/config');
+            fs.writeFileSync(
+              'modules/core/client/config/core.client.routes.js',
+              "'use strict';\n\n" +
+              "//Setting up route\n" +
+              "angular.module('core').config(['$stateProvider',\n" +
+              "  function($stateProvider) {\n" +
+              "    // Core state routing\n" +
+              "    $stateProvider.\n" +
+              "      state('core', {\n" +
+              "        url: '/fooroute',\n" +
+              "        templateUrl: 'modules/core/client/views/core.client.view.html'\n" +
+              "      });\n" +
+              "  }\n" +
+              "]);\n",
+              'utf8'
+            )
+          })
+          .on('end', function () {
+            done();
+          });
+      });
+
+      it('should generate an angular view file', function () {
+        assert.file('modules/core/client/config/core.client.routes.js');
+        assert.file('modules/core/client/views/bar.client.view.html');
+
+        assert.fileContent('modules/core/client/config/core.client.routes.js',
+          "'use strict';\n\n" +
+          "//Setting up route\n" +
+          "angular.module('core').config(['$stateProvider',\n" +
+          "  function($stateProvider) {\n" +
+          "    // Core state routing\n" +
+          "    $stateProvider.\n" +
+          "      state('bar', {\n" +
```

Comment:
If @ilanbiala's comment below on line 187 holds true, than it should be the same up here as well.